applications/machine_learning/README.rst

```diff
@@ -69,7 +67,7 @@ The labels that are assigned by the machine learning model are specific to the g
 
 By default, the application uses pretrained machine leaning models deployed in `Edge Impulse studio`_:
 
-* Both Thingy:52 and Thingy:53 use the `NCS hardware accelerometer machine learning model`_.
+* Thingy:53 use the `NCS hardware accelerometer machine learning model`_.
```

Comment:
s/use/uses
But please do not rerun CI because of it. Let's fix this one one earliest occasion after the PR is merged.

Reply:
There are some minor doc fixes for which I will create a doc PR later.